source/_integrations/rflink.markdown

```diff
@@ -89,10 +89,12 @@ rflink:
 
 TCP mode allows you to connect to an RFLink device over a TCP/IP network. This is useful if placing the RFLink device next to the HA server is not optimal or desired (eg: bad reception).
 
-To expose the USB/serial interface over TCP on a different host (Linux) the following command can be used:
+To expose the USB/serial interface over TCP on a different host (Linux) the following command can be used. The arguments are separated by spaces, further info on all arguments can be found for example [on the Debian manpages](https://manpages.debian.org/stretch/socat/socat.1.en.html).
+- `/dev/ttyACM0,b57600,rawer` specifies the device location, a `b57600` 57600 baud rate and `rawer` causes socat to ignore control sequences sent via the port (i.e. it makes socat pass all information 'rawest form', rather than picking up control characters such as control-C which would close socat).
```

Comment:
```suggestion
- `/dev/ttyACM0,b57600,rawer` specifies the device location, a `b57600` 57600 baud rate, and `rawer` causes socat to ignore control sequences sent via the port (for example, it makes socat pass all information 'rawest form', rather than picking up control characters such as control-C which would close socat).
```